resources/test-invoker.mjs

```diff
@@ -24,7 +24,24 @@ export class TimerTestInvoker extends TestInvoker {
     }
 }
 
-export class RAFTestInvoker extends TestInvoker {
```

Comment:
I'd prefer removing the TimerTestInvoker entirely since we've moved to RAF-based measurement in 3 and we haven't seen any issue with it.

Reply:
removed!